charts/fluentd-elasticsearch/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: fluentd-elasticsearch
-version: 4.3.2
+version: 4.3.3
```

Comment:
new feature needs update of minor version (4.4.0)

Reply:
Bumped minor version